### PR TITLE
[en]: Update best-practice/tail-based-sampling

### PR DESCRIPTION
**Pull request by WANG0608GitHub**
- Update English based on Chinese

- MTPE

### CODE REVIEW

**Comment by WANG0608GitHub** [comment]
see preview: https://deploy-preview-5618--daocloud-docs.netlify.app/en/insight/best-practice/tail-based-sampling

File: docs/en/docs/insight/best-practice/tail-based-sampling.md [hunk 1]
```diff
@@ -1,245 +1,338 @@
...
 
-### Deploy OTEL COL Component with LB Capability
+It is important to use consistent terminology when discussing sampling. A Trace or Span is considered **sampled** or **unsampled**:
 
-Refer to the following YAML to deploy the component.
+- **Sampled**: A Trace or Span that is processed and stored. It is chosen by the sampler to represent the overall data, so it is considered **sampled**.
+- **Unsampled**: A Trace or Span that is not processed or stored. Because it was not selected by the sampler, it is considered **unsampled**.
 
-??? note "Click to view deployment configuration"
+## What Are the Sampling Methods?
```
Comment (windsonsea) [inline docs/en/docs/insight/best-practice/tail-based-sampling.md:29]:
```suggestion
## What Are the Sampling Options?
```

File: docs/en/docs/insight/best-practice/tail-based-sampling.md [hunk 2]
```diff
@@ -1,245 +1,338 @@
...
+- Efficient
+- Can be performed at any point in the tracing collection pipeline
+
+The main drawback of head sampling is that it cannot make sampling decisions based on the data within
+the entire Trace. This means that while head sampling is effective as a blunt instrument, but it is
+completely insufficient for sampling strategies that must consider information from the entire system.
+For example, head sampling cannot ensure that all Traces with errors are sampled. For this, you need tail
+sampling.
+
+### Tail Sampling — Recommended Approach
```
Comment (windsonsea) [inline docs/en/docs/insight/best-practice/tail-based-sampling.md:53]:
```suggestion
### Tail Sampling (Recommended)
```

**Review by windsonsea — COMMENTED** [review]
尽量留用 native 的语句，不用再重新翻译：
https://opentelemetry.io/docs/concepts/sampling/

**Review by windsonsea — APPROVED** [review]

**Pull request merged**